modules/docs/mdx/5.0-MIGRATION-GUIDE.mdx

```diff
@@ -823,8 +825,9 @@ interface MyButtonProps extends ButtonProps {
 
 ### Card
 
-Card is now a [compound component](./COMPOUND_COMPONENTS.mdx) composed of a `Card.Body` and an
-optional `Card.Heading`. This allows direct access to the heading and body elements.
+Card is now a [compound component](/getting-started/for-developers/resources/compound-components/)
```

Comment:
This change will keep Storybook links working. This will break the link in Github, but I think that's okay since the document is not easy to find while looking at the file in Github.

Reply:
This makes me a bit nervous. We have linked to github docs a lot, so any historical references (old slack posts, etc.) will have broken links after this. At the very least, we should swap these to fully qualified URLs once the site is live.

Reply:
We've broken URLs in the past when we moved these component documents from the root of the project to the `docs` package. It is possible to make links work for Github (interlinking), Storybook, and the Canvas Site, but complexity rises significantly. We have to drop relative links between Github docs at the least since the URLs won't exist on the Canvas Site, but then we run into a issue with active branches unless we always assume `master`.

I think it's time to drop support for Github doc rendering. We used Github's interlinking because we didn't have a good way to link between Storybook docs. The Canvas Site url scheme presents a better way of linking between documents and, with this change, works on both the Canvas Site and Storybook.